src/commands/transaction/send_meta_transaction/sign_as/mod.rs

```diff
@@ -73,11 +73,16 @@ impl RelayerAccountId {
     fn input_relayer_account_id(
         context: &super::SendMetaTransactionContext,
     ) -> color_eyre::eyre::Result<Option<crate::types::account_id::AccountId>> {
+        let mut relayer_account_id: crate::types::account_id::AccountId =
+            CustomType::new("What is the relayer account ID?").prompt()?;
+
+        if context.global_context.offline {
+            return Ok(Some(relayer_account_id));
+        }
```

Comment:
Let's move the prompt back into the loop. It is not a problem that `offline` would be checked in every loop instance, but copy-pasted message asking relayer account id is easy to miss and have two different messages in the future.

Reply:
Done